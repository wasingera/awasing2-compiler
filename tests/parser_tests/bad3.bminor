bad_parse integer = !false;
