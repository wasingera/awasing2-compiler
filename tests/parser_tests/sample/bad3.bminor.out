parse error: syntax error
Parse failed: "hello world"
